### ISSUE TITLE
any better video inpainting module than STTN?

### ISSUE THREAD

**Issue by dragen1860** [geekyutao/Inpaint-Anything#72]
Dear author:
   Thanks for sharing the open source video inpainting tool, it works pretty useful.
   When talking about improving the video inpainting result, i want to know, any robust and better quality video inpainting module can be used other than STTN? i tried some paper such as FGT, Fuseformer， but it does not generalize well as expected. Thank you.

**Comment by geekyutao** [comment]
> 

We actually tested several video inpainting models (including CVPR22 works). Unfortunately, a few of them performed good. STTN is not new but relatively robust.

**Comment by dragen1860** [comment]
> 

Yes, I agree. STTN AND DSTT is relatively robust and generalize well. other SOTA performs well on the test set but when inference on in-the-wild phone video, it performs very bad.
